client/extensions/woocommerce/state/wc-api/shipping-methods/actions.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal dependencies
+ */
+import request from '../request';
+import { setError } from '../../site/status/wc-api/actions';
```

Comment:
Done in https://github.com/Automattic/wp-calypso/pull/14610/commits/a647fe757df73b015ec06853eebf002be12f7cc8 . I had to add the `client/extensions/` root path to the test runner commands because tests don't use Webpack.